docs/eventing/README.md

```diff
@@ -39,9 +39,24 @@ generic interfaces that can be implemented by multiple Kubernetes resources:
    returned events may be further processed in the same way that events from an
    external event source are processed.
 
+### Event brokers and triggers
+
+As of v0.5, Knative Eventing defines Broker and Trigger objects to make it
+easier to filter events.
+
+A Broker provides a bucket of events which can be selected by attribute. It
+receives events and forwards them to subscribers defined by one or more matching
+Triggers. Each namespace has a default Broker, and for most use cases, no
```

Comment:
The namespace only has a default Broker if it is annotated. If it is not annotated, no default Broker is created (although users are free to create it themselves).

Reply:
Ah, okay. Deleted that sentence altogether.